src/device-registry/routes/v2/test/ut_devices.js

```
@@ -1,182 +1,298 @@
+      json: sinon.spy(),
+      status: sinon.stub().returns(res),
+      send: sinon.spy(),
+    };
   });
 
   afterEach(() => {
     sinon.restore();
   });
+});
+
+const sinon = require("sinon");
```

Biome

[error] Shouldn't redeclare 'sinon'. Consider to delete it or rename it.

'sinon' is defined here:


(lint/suspicious/noRedeclare)